Dockerfile

```diff
@@ -7,27 +7,32 @@ ARG CONCURRENCY=4
 # setup the ondemand repositories
 RUN dnf -y install https://yum.osc.edu/ondemand/latest/ondemand-release-web-latest-1-6.noarch.rpm
 
+# setup yarn repository
+RUN curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo && \
+    rpm --import https://dl.yarnpkg.com/rpm/pubkey.gpg
 # install all the dependencies
 RUN dnf -y update && \
     dnf -y module enable nodejs:12 ruby:2.7 && \
     dnf install -y \
-        file \
-        lsof \
-        sudo \
-        gcc \
-        gcc-c++ \
-        git \
-        patch \
-        ondemand-gems \
-        ondemand-runtime \
-        ondemand-build \
-        ondemand-apache \
-        ondemand-ruby \
-        ondemand-nodejs \
-        ondemand-python \
-        ondemand-dex \
-        ondemand-passenger \
-        ondemand-nginx && \
+    yarn \
+    file \
+    lsof \
+    sudo \
+    gcc \
+    gcc-c++ \
+    git \
+    patch \
+    yarn \
+    ondemand-gems \
+    ondemand-runtime \
+    ondemand-build \
+    ondemand-apache \
+    ondemand-ruby \
+    ondemand-nodejs \
+    ondemand-python \
+    ondemand-dex \
+    ondemand-passenger \
+    ondemand-nginx && \
```

Comment:
for the packages fix  the indentation to be like before - the arguments to `dnf install -y` are indented further on purpose but that was lost here